deployer/scripts/install.sh

```diff
@@ -373,6 +373,10 @@ function generate_objects() {
   generate_kibana
   generate_curator
   generate_fluentd
+
```

Comment:
yes, and the `oc import-image` lines too. this complicates the concept of
an upgrade but i think in the end, if everything is scaled down and then we
get everything up to the latest before scaling up... it won't matter that
we're skipping updating imagestreams that don't end up being relevant. we
should probably even delete the imagestreams (if they have the template
label) in the upgrade to reduce confusion.

On Fri, Jun 3, 2016 at 9:37 AM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployer/scripts/install.sh
> <https://github.com/openshift/origin-aggregated-logging/pull/157#discussion_r65707745>
> :
>
> > @@ -373,6 +373,10 @@ function generate_objects() {
> >    generate_kibana
> >    generate_curator
> >    generate_fluentd
> > +
>
> should that also be removed from
> https://github.com/ewolinetz/origin-aggregated-logging/blob/644af15b3d1722be7df576136917963616194373/deployer/scripts/upgrade.sh#L431
> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/157/files/644af15b3d1722be7df576136917963616194373#r65707745>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz-vflngkBxiMAg73E8H3rhNh6OTjxks5qIC4HgaJpZM4IpZyZ>
> .
>